photon-ml/src/main/scala/com/linkedin/photon/ml/optimization/game/GLMOptimizationConfiguration.scala

```diff
@@ -38,16 +37,23 @@ protected[ml] case class GLMOptimizationConfiguration private (
     regularizationType: RegularizationType = RegularizationType.L2) {
 
   override def toString: String = {
-    s"maxNumberIterations: $maxNumberIterations\t" +
-      s"convergenceTolerance: $convergenceTolerance\t" +
-      s"regularizationWeight: $regularizationWeight\t" +
-      s"downSamplingRate: $downSamplingRate\t" +
-      s"optimizerType: $optimizerType\t" +
+    s"maxNumberIterations: $maxNumberIterations," +
+      s"convergenceTolerance: $convergenceTolerance," +
+      s"regularizationWeight: $regularizationWeight," +
+      s"downSamplingRate: $downSamplingRate," +
+      s"optimizerType: $optimizerType," +
```

Comment:
It's okay, we can leave it for now. The upcoming PR for the OptimizationProblem refactor modifies this part, I'll make a note of it when the PR goes out to draw your attention to it